dev-packages/node-integration-tests/suites/express/tracing/updateName/test.ts

```diff
@@ -29,11 +29,11 @@ describe('express tracing', () => {
 
     // Also calling `updateName` AND setting a source doesn't change anything - Otel has no concept of source, this is sentry-internal.
     // Therefore, only the source is updated but the name is still overwritten by Otel.
-    test("calling `span.updateName` and setting attribute source doesn't update the final name in express but it updates the source", async () => {
+    test('calling `span.updateName` and setting attribute source updates the final name in express', async () => {
       const runner = createRunner(__dirname, 'server.js')
         .expect({
           transaction: {
-            transaction: 'GET /test/:id/span-updateName-source',
+            transaction: 'new-name',
```

Comment:
this is actually... fixed here now? Not quite sure why/how, but apparently some timing stuff is better now?

Reply:
this indicates that the retroactive span name extraction is no longer use. Which makes sense to me if we now set a good name right from the start! 

(I think this is fine btw. Actually leaves less room for user error in case they don't use the `Sentry.updateSpanName` helper)